siphon/simplewebservice/spc.py

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) 2019 Siphon Contributors.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+"""Reading Storm Prediction Center Data.
+
+======================================
+This program pulls data from the Storm Prediction
+Center's Archive of data that goes back to the 1950s.
+Weather events that are available are
+hail, wind, and tornados.
+
+"""
+
+import pandas as pd
+
+
+class SpcData:
```

Comment:
`SpcData` should also use `HTTPEndPoint`, in the way that `NDBC` does. In this case, we would use the methods from `HTTPEndPoint` to read the data from a URL, and pass that to pandas `pd.read_csv` by wrapping the `HTTPEndPoint` return in a `StringIO`.  Again,  @dopplershift @zbruick, is that something we refactor later, given the summer timeline?